store/job_store.go

```diff
@@ -63,3 +91,28 @@ func (ds *DataStore) GetJob(ctx context.Context, id string) (models.Job, error)
 	log.Event(ctx, "getting job from job store", log.Data{"Job details: ": JobsMap[id]})
 	return job, nil
 }
+
+//GetJobs gets a list of Job resources from the JobsMap. It will put all the Job resources into a list, sorted by their
+//last_updated time values, and return the list.
+func (ds *DataStore) GetJobs(ctx context.Context) (models.Jobs, error) {
+	log.Event(ctx, "getting list of jobs")
```

Comment:
logs in this file are missing the severity level. This will not be optional in the future (via log.go library).

Please include log.INFO in the event logs (line 98, 104, 115)

Reply:
I've added it now